### ISSUE TITLE
Failed to import obfuscated modules with docstrings

### ISSUE THREAD

**Issue by joonis** [dashingsoft/pyarmor#1013]
If an obfuscated module contains a docstring, I get the following error:

```
Python 3.8.10 (default, Mar 13 2023, 10:26:41) 
[GCC 9.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import mypackage
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "</home/.../mypackage/__init__.py>", line 3, in <module>
  File "<mypackage.__init__>", line 2, in <module>
TypeError: <lambda>() takes 0 positional arguments but 1 was given
```

The package has been obfuscated with Pyarmor 8.0.17:

`pyarmor gen -i -O ./build/pyarmor mypackage`

**Comment by jondy** [comment]
Reproduced, it will be fixed in next release.

**Comment by jondy** [comment]
Fixed in 8.0.18
